projects/kit/components/marker-icon/marker-icon.component.ts

```diff
@@ -17,7 +17,7 @@ import {TuiMarkerIconMode} from '@taiga-ui/kit/types';
 import {Observable} from 'rxjs';
 
 @Component({
-    selector: `tui-marker-icon`,
+    selector: `tui-marker-icon, [tuiMarkerIcon]`,
```

Comment:
Let's make it more concrete `a[tuiMarkerIcon], button[tuiMarkerIcon]`

Reply:
fixed